query/security/wcpm/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,87 +15,90 @@
 
     <reference-list id="authHandlers"
                     interface="org.codice.ddf.security.handler.api.AuthenticationHandler"
-                    availability="optional"/>
+                    availability="optional">
+        <reference-listener bind-method="bindValue" unbind-method="unbindValue" ref="authTypePoller"/>
+    </reference-list>
 
     <reference-list id="realms"
                     interface="org.apache.karaf.jaas.config.JaasRealm"
-                    availability="optional"/>
+                    availability="optional">
+         <reference-listener bind-method="bindValue" unbind-method="unbindValue" ref="realmPoller"/>
+    </reference-list>
 
+    <bean id="authTypePoller" class="org.codice.ddf.admin.security.wcpm.AuthTypesPoller">
+        <property name="authHandlers" ref="authHandlers"/>
+        <property name="descriptionMap">
```

Comment:
❓ So any auth types that we find that are not included here will just have blank descriptions?